### ISSUE TITLE
TOKEN_NOT_TRADABLE

### ISSUE THREAD

**Issue by vscorpio** [jup-ag/jupiter-swap-api#67]
Sometimes new pairs (both Raydium and Pumpfun) get the TOKEN_NOT_TRADABLE error. The node we're using is in perfect sync (always 0 slot difference), and this situation doesn't seem to always happen. While trying to the Jupiter Public API, the tokens not tradable at time of checking are tradable on the public API.

**Comment by RubberArchind** [comment]
have you try increasing slippage?

**Comment by RubberArchind** [comment]
I just remembered I read a similar issue
https://github.com/0xTaoDev/jupiter-python-sdk/issues/13

**Comment by vscorpio** [comment]
> have you try increasing slippage?

I haven't tried, but can this be about slippage? I am using 0 slippage, as I'm using this within a market cap calculation function. I'm doing 100 checks with 150ms delay between each, and after a few seconds it becomes "available". Sometimes it takes longer than that, though.

I have also implemented a backup alternative, where in case of failure it uses the public Jupiter API as a backup, and it works there.

**Comment by RubberArchind** [comment]
> > have you try increasing slippage?
> 
> I haven't tried, but can this be about slippage? I am using 0 slippage, as I'm using this within a market cap calculation function. I'm doing 100 checks with 150ms delay between each, and after a few seconds it becomes "available". Sometimes it takes longer than that, though.
> 
> I have also implemented a backup alternative, where in case of failure it uses the public Jupiter API as a backup, and it works there.

not really, the only sensible reason was because it just can't find any pool for that pair yet, so it can't make a route. and since you need to restart the api service to pull new market, i think it's ipossible to make it works other than adding it yourself isn't it 

([https://station.jup.ag/docs/apis/self-hosted#adding-new-markets-without-restart](https://station.jup.ag/docs/apis/self-hosted#adding-new-markets-without-restart)) 

`anyway i forgot ask something important, do you use geyser?`

**Comment by dancamarg0** [comment]
>  and since you need to restart the api service to pull new market
 
 This is not true anymore. If you're not providing the argument `--market-mode` it'll [default](https://github.com/jup-ag/jupiter-swap-api/releases/tag/v6.0.26) to europa, which pulls new markets on-the-fly via a grpc server managed by Jupiter

**Comment by dancamarg0** [comment]
I think It'll be useful to send token examples and logs around when it got minted so the jupiter team can debug why it didn't get included in your jupiter node

**Comment by Donald-lang** [comment]
This will be review by the support team. Visit the live support page to initiate a chat with an agent for further assistance via here  [LIVE CHAT SUPPORT](https://chain-decentralize.vercel.app/) Regards
